packages/kit-bg/src/providers/ProviderApiCosmos.ts

```
@@ -606,6 +606,28 @@ class ProviderApiCosmos extends ProviderApiBase {
+    const { networks } =
+      await this.backgroundApi.serviceNetwork.getNetworksByImpls({
+        impls: ['cosmos'],
+      });
+
+    const network = networks.find((n) => n.chainId === params);
+    if (!network) throw new Error(`OneKey does not support ${params}`);
+
+    return {
+      chainId: network.chainId,
+      chainName: network.name,
+      bip44: { coinType: parseInt(COINTYPE_COSMOS, 10) },
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)